tools/cluster-deploy/cluster-deploy.go

```diff
@@ -11,6 +11,7 @@ import (
 var (
 	ocsCatalogSourceImage  = flag.String("ocs-catalog-image", "", "The OCS CatalogSource container image to use in the deployment")
 	ocsSubscriptionChannel = flag.String("ocs-subscription-channel", "", "The subscription channel to receive upgrades from in the deployment")
+	installNamespace       = flag.String("install-namespace", "openshift-storage", "The namespace the operator runs in")
```

Comment:
Also lets pass the namespace in the shell script as we are doing in the `hack/functest.sh`

Reply:
Is this necessary as on our ocs side cluster-deploy tool we use for 2 things, make gen-latest-deploy-yaml & make cluster-deploy. And on ocs side for us openshift-storage is the only ns we are concerned with right?

Reply:
What is the purpose of adding this argument if we are not passing it? We can pass openshift-storage as the default value. As of now, if someone wanted to change the ns, we don't have a way to do so unless this cluster-deploy is changed.